types/types.go

```diff
@@ -741,6 +741,17 @@ func ValidateReplicaCount(count int) error {
 	return nil
 }
 
+func ValidateMinNumberOfBackingIamgeCopies(number int) error {
+
+	definition, _ := GetSettingDefinition(SettingNameDefaultMinNumberOfBackingImageCopies)
```

Comment:
Check the returned boolean value before accessing definition.

Reply:
fixed.